<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monsterForm</class>
 <widget class="QWidget" name="monsterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_Del">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_Monsters">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>221</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLineEdit" name="txt_Name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>HP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>MP</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Attack</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Defense</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>EXP</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="num_ATK">
      <property name="minimum">
       <number>-9999</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="num_DEF">
      <property name="minimum">
       <number>-9999</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="num_EXP">
      <property name="minimum">
       <number>-9999</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="num_MP">
      <property name="minimum">
       <number>-9999</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>img</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="num_HP">
      <property name="minimum">
       <number>-9999</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txt_IMG"/>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btn_IMG">
      <property name="text">
       <string>File...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_Add">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addSkill">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Skill</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rmSkill">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Skill</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_Mskills">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>141</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <zorder>lst_Mskills</zorder>
  <zorder>btn_Del</zorder>
  <zorder>lst_Monsters</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>btn_Add</zorder>
  <zorder>btn_addSkill</zorder>
  <zorder>btn_rmSkill</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
